src/definitions/internet.ts

```diff
@@ -1,5 +1,17 @@
 import { allOf } from './utils';
 
+export type EmojiGroups =
+  | 'smileys'
```

Comment:
```suggestion
  | 'smiley'
```

Can you keep singular form for consistency?

Reply:
Fixed here: https://github.com/faker-js/faker/pull/504/commits/6badfaaa55667c694d80753bbb63698be463229a